### ISSUE TITLE
usbfs: process uuu did not claim interface 0 before use

### ISSUE THREAD

**Issue by nsjodk** [NXPmicro/mfgtools#70]
I get the following messages in dmesg:

hid-generic 0003:15A2:0080.002A: hiddev2,hidraw4: USB HID v1.10 Device [Freescale SemiConductor Inc  SE Blank 6ULL] on usb-0000:00:14.0-3/input0
usb 1-3: usbfs: process 17860 (uuu) did not claim interface 0 before use

**Comment by nxpfrankli** [comment]
Does it impact function or warning message?
what's your libusb version? which linux distribution do you running?

**Comment by raveslave** [comment]
related to issue #72 I had 'dmesg --follow' running while trying to update the imx8 evkit -> emmc
latest uuu built from source on Ubuntu 18.04
dependencies via apt-get official

**Comment by raveslave** [comment]
@nxpfrankli was this fixed for Ubuntu? Using the 1.2.135 release

dmesg:
```
[ 4156.049696] usb 1-1: New USB device found, idVendor=1fc9, idProduct=0134
[ 4156.049698] usb 1-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[ 4156.049699] usb 1-1: Product: SE Blank M845S 
[ 4156.049700] usb 1-1: Manufacturer: NXP      SemiConductors Inc 
[ 4156.060513] hid-generic 0003:1FC9:0134.000B: hiddev0,hidraw0: USB HID v1.00 Device [NXP      SemiConductors Inc  SE Blank M845S ] on usb-0000:00:1d.7-1/input0
```

uuu
```
mfgtools/uuu$ ./uuu ../../L4.14.78_1.0.0_ga_images_MX8MQEVK/uuu.auto
uuu (Universal Update Utility) for nxp imx chips -- libuuu_1.2.135-0-gacaf035

Success 0    Failure 0                                                                                                         
                                                                                                                                
1:1      1/ 1 [=================100%=================] SDP: boot -f imx-boot-imx8mqevk-sd.bin-flash_evk       
```

then nothing...

**Comment by raveslave** [comment]
ehrm... realized I tried to flash 8MQ onto a 8MM EV-Kit
a nice thing would be an error message that compares the USB product-id and simply report if the build-tree mismatches what you're trying to flash

**Comment by nxpfrankli** [comment]
L4.14.78_1.0.0_ga_images_MX8MQEVK/imx-boot-imx8mqevk-sd.bin-flash_evk have not included chip information.

**Comment by tobiz** [comment]
I've got the same problem with my Eaton 5P 850 running IPP on Ubuntu 18.04.  I keep getting dmesg such as :

[ 9288.709526] usb 1-1: usbfs: process 18322 (Eaton-IPP) did not claim interface 0 before use
[ 9299.567961] usb 1-1: usbfs: process 4375 (usbhid-ups) did not claim interface 0 before use

Ignore the process numbers and other data.
The Eaton 5p 850 is connected via its USB port

I've also got NUT installed and can access the 5P ok. 

But IPP will connect and then fail with comms error but after some hours seems to reestablish the connection, some times.

lsusb gives:
Bus 001 Device 002: ID 0463:ffff MGE UPS Systems UPS
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

dmesg gives
dmesg |grep 0463
[    2.499855] usb 1-1: New USB device found, idVendor=0463, idProduct=ffff
[    9.860456] hid-generic 0003:0463:FFFF.0001: collection stack overflow
[    9.860462] hid-generic 0003:0463:FFFF.0001: item 0 1 0 10 parsing failed
[    9.860480] hid-generic: probe of 0003:0463:FFFF.0001 failed with error -22
where's Bus 001 Device 002 gone?  None of this looks right!

If I do:
sudo upsdrvctl stop
Network UPS Tools - UPS driver controller 2.7.4

If I then do:
sudo upsdrvctl start
I get:
Network UPS Tools - UPS driver controller 2.7.4
Network UPS Tools - Generic HID driver 0.41 (2.7.4)
USB communication driver 0.33
Using subdriver: MGE HID 1.39

I'm no expert in this area but it looks like the usbhid-generic and usbhid-ups are trying to both be allocated.


If anyone can help on this it'd most welcome.
